src/containers/Dashboard.js

```diff
@@ -0,0 +1,163 @@
+    const showDashboard =
+      this.state.display === Display.LOADING ||
+      this.state.display === Display.ERROR
+        ? 'none'
+        : 'block';
+    return (
+      <Container>
+        <Row className='justify-content-center mt-5'>
+          <h1>Dashboard App</h1>
+        </Row>
+        <Row className='justify-content-center mb-5 mt-5'>
+          <Button
+            variant='info'
+            size='sm'
+            onClick={this.generateLocalData}
+            style={{ margin: 10 }}
+            active={this.state.display === Display.FRONTEND}
+          >
+            Generate data in frontend
+          </Button>
+
+          <Button
+            variant='info'
+            size='sm'
+            onClick={this.getRemoteData}
+            style={{ margin: 10 }}
+            active={this.state.display === Display.BACKEND}
+          >
+            Generate data in backend
+          </Button>
+
+          <Button
+            variant='info'
+            size='sm'
+            onClick={this.fetchStoredData}
+            style={{ margin: 10 }}
+            active={this.state.display === Display.DB}
+          >
+            Fetch data from database
+          </Button>
+        </Row>
+        <Container style={{ display: showDashboard }}>
+          <Row style={{ marginBottom: 100 }}>
+            <Col className='col-4'>
+              <Chart type='doughnut' data={this.state.chartData} />
+            </Col>
+            <Col className='col-4'>
+              <Chart type='pie' data={this.state.chartData} />
+            </Col>
+            <Col className='col-4'>
+              <Chart type='bar' data={this.state.chartData} />
+            </Col>
+          </Row>
+          <Row>
+            <Col className='col-4'>
+              <Chart type='line' data={this.state.chartData} />
+            </Col>
+            <Col className='col-4'>
+              <Chart type='polarArea' data={this.state.chartData} />
+            </Col>
+            <Col className='col-4'>
+              <Chart type='horizontalBar' data={this.state.chartData} />
+            </Col>
+          </Row>
+        </Container>
+        <Container className={showError}>
+          <Alert variant='danger'>Oh snap! You got an error!</Alert>
+        </Container>
+        <Container fluid className={showLoading}>
+          <Spinner animation='border' variant='info' className='mr-2' />
+          <span className='pl-1'>Loading...</span>
+        </Container>
+      </Container>
+    );
+  }
+
+  /* Generate random chart data */
+  generateLocalData() {
+    const newdata = Object.assign({}, this.state.chartData);
+    newdata.datasets[0].data = [
+      Math.floor(Math.random() * 100 + 1),
+      Math.floor(Math.random() * 100 + 1),
+      Math.floor(Math.random() * 100 + 1),
+    ];
+    this.setState({ display: Display.FRONTEND });
+    this.setState({ chartData: newdata });
+  }
+
+  /* Generate random chart data from backend*/
```

Comment:
```suggestion
  /** Generate random chart data from backend */
```
To make it a docstring.

Reply:
Same applies to the comments above.

Reply:
Fixed in all files, thanks.